includes/formatters/users.php

```diff
@@ -75,6 +75,25 @@ function user_pass( $user, $formatter ) {
 	return $user;
 }
 
+/**
+ * Generate a random password. Support both PHP 5.6 and PHP 7.0+.
+ *
+ * @return string
+ */
+function generate_random_password() {
+	$password = 'password';
```

Comment:
Why do we set a default password here?

What scenario would we have where both function_exists( 'random_bytes' ) is false and function_exists( 'mcrypt_create_iv' )  is false?

Reply:
There isn't any. This is an unnecessary safety step, a fix is incoming.